en/micro-integrator/docs/references/connectors/connector-usage.md

```diff
@@ -0,0 +1,159 @@
+
+## Performance tuning and monitoring
+
+SaaS connectors use HTTP/HTTPS protocol to communicate. They use WSO2 EI mediation engine itself. Hence HTTP protocol related tunings are applied. 
+
+Technology connectors use protocols that are custom. Thus their tuning needs to be done at the connector itself. All connection related tunings are present in the form you get when you create a new connection for the connector. For the older connectors configurations will be present in init operation. 
+
+Please refer to the reference documentation of the connector for details. 
+
+## Troubleshooting
+
+### Enable detailed logging 
+
+* Connector implementations will have DEBUG and TRACE level logs. You can enable them to see in detail what is going on with the connector. 
+
+* See [Configuring Log4j2 Properties section of the documentation](../../administer-and-observe/logs/configuring_log4j_properties/) on how to enable DEBUG logs specifically for a Java package and on how to view the logs.
+
+* To get the package name of the connector implementation, refer to the [How to contribute section of the overview page of connector documentation](connectors-overview/#contribute-to-the-connector-project). 
+
+### Enable wire logging 
+
+For SaaS connectors which use HTTP transport of WSO2 EI, developers can enable wire logs to see details of the messages that are sent from WSO2 EI to the backend service and the response sent back to WSO2 EI. This is useful to check the exact message that is sent out by the connector to the backend service. See [documentation on monitoring wire logs](../../administer-and-observe/logs/monitoring_logs/#wire-logs) for instructions on how to enable wire logs. 
+
+### Mediation debug 
+
+WSO2 Integration Studio provides debugging capabilities. You cannot use mediation debugging to debug templates packaged inside a connector. However, you can use it to check the following. 
+
+* Whether you are passing the correct message into the connector operation.
+* Whether your input parameters for connector operations contain the expected values.
+* What is the response message after using connector cooperation.
+
+Please refer to [the Debugging Mediation documentation](../../develop/debugging-mediation/) for instructions on how to use mediation debugging. 
+
+### Debugging connector code 
+
+You can get the source code of WSO2 EI connector and remotely debug it with your scenario to find out issues. Refer to [the How to contribute” section of the overview page of connector documentation](connectors-overview/#contribute-to-the-connector-project), get the GitHub repository, clone it, checkout to relevant version, and debug. It is open source!
```

Comment:
[the How to contribute” section of the overview page of connector documentation] Formatting error. 

[the "How to contribute” section of the overview page of connector documentation]?